js/form.js

```diff
@@ -0,0 +1,134 @@
+console.clear();
+
+const form = document.querySelector('[data-js="question-form"]');
+const mainContainer = document.querySelector("main");
+const inputQuestion = document.querySelector('[data-js="question-form-input"]');
+const characterCounterQuestion = document.querySelector(
+  '[data-js="character-counter-question"]'
+);
+const inputAnswer = document.querySelector('[data-js="answer-form-input"]');
+const characterCounterAnswer = document.querySelector(
+  '[data-js="character-counter-answer"]'
+);
+
+inputQuestion.focus();
+
+// This is the function for the character counters
+
+function textfieldInput(input, characterCount) {
+  characterCount.textContent =
+    input.getAttribute("maxlength") - input.value.length;
+}
+
+inputQuestion.addEventListener("input", () => {
+  textfieldInput(inputQuestion, characterCounterQuestion);
+});
+
+inputAnswer.addEventListener("input", () => {
+  textfieldInput(inputAnswer, characterCounterAnswer);
+});
+
+// Function to add '#'
+
+function hashtag(data) {
+  const input = data.split(" ").join("");
```

Comment:
den Tipp hattest Du mir ja weitergegeben. Ich habe es noch mit toLowerCase ergänzt, falls Du das auch willst...

```suggestion
  const input = data.split(" ").join("").toLowerCase();
```